data/crac-io/crac-io-json/src/test/java/com/farao_community/farao/data/crac_io_json/CracImportExportTest.java

```diff
@@ -216,5 +236,15 @@ public void cracTest() {
         OnFlowConstraint onFlowConstraint2 = (OnFlowConstraint) importedCrac.getHvdcRangeAction("hvdcRangeId2").getUsageRules().get(0);
         assertEquals(Instant.PREVENTIVE, onFlowConstraint2.getInstant());
         assertSame(importedCrac.getCnec("cnec3prevIdBis"), onFlowConstraint2.getFlowCnec());
+
+        assertEquals(1, crac.getNetworkAction("switchPairRaId").getElementaryActions().size());
+        assertTrue(crac.getNetworkAction("switchPairRaId").getElementaryActions().iterator().next() instanceof SwitchPair);
+        SwitchPair switchPair = (SwitchPair) crac.getNetworkAction("switchPairRaId").getElementaryActions().iterator().next();
+        assertEquals("to-open", switchPair.getSwitchToOpen().getId());
+        assertEquals("to-open", switchPair.getSwitchToOpen().getName());
+        assertEquals("to-close", switchPair.getSwitchToClose().getId());
+        assertEquals("to-close-name", switchPair.getSwitchToClose().getName());
+
+        assertEquals(2, crac.getNetworkAction("switchPairRaId2").getElementaryActions().size());
```

Comment:
test importedCrac instead of crac variable

Reply:
true ! fixed